docs/tutorials/getting-started/rust.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Rust SDK and Axum
+description: Getting Started with the OpenFeature Rust SDK and Axum (with Docker example)
+---
+
+import WhyDefaultContent from '@site/src/components/custom/tutorial/why-default-content.mdx';
+import FlagdContent from '@site/src/components/custom/tutorial/flagd-content.mdx';
+import FlagdChangeContent from '@site/src/components/custom/tutorial/flagd-change-content.mdx';
+
+# Getting Started with the OpenFeature Rust SDK and Axum
+
+## Introduction
+
+This tutorial explains how to integrate the OpenFeature Rust SDK into an Axum application using the flagd provider. In this guide you will learn how to:
+
+- Create a new Rust project with Cargo
+- Update dependencies to include Docker-based flagd configuration
+- Set up an Axum server with a custom AppState that holds a FlagdProvider
+- Evaluate a feature flag and return a custom message with diff markers to highlight changes
+
+## Requirements
+
+- Rust stable (1.80.2 or later recommended)
+- Cargo
+- Docker (to run the flagd service)
+- A valid flagd configuration file (e.g. `flags.flagd.json`)
+
+## Walk-through
+
+### Step 1: Create a new Rust project
+
+```bash
+cargo new axumstart
+```
+
+Then navigate to the project directory:
+
+```bash
+cd axumstart
+```
+
+### Step 2: Update Cargo.toml dependencies
+
+In your Cargo.toml add the following dependencies:
+
+```toml
+[dependencies]
+axum = "0.6"
+tokio = { version = "1", features = ["full"] }
+serde_json = "1.0"
+tracing = "0.1"
+open-feature = "^0"
+open-feature-flagd = "^0"
+```
```

Comment:
I had to include `openssl = { version = "0.10", features = ["vendored"] }` for it to work for me but perhaps it was just an issue with my setup.